src/shared/components/input/Input.scss

```diff
@@ -1,23 +1,60 @@
 @import 'src/styles/common';
 
 .input {
-  display: inline-block;
-  width: 100%;
+  &::placeholder {
+    color: $dark-grey;
+    font-weight: $light;
+  }
 
   &:focus,
   &:active {
     outline: none;
+    color: $black;
+    font-weight: $normal;
   }
+}
 
-  &::placeholder {
-    color: $black;
-    font-weight: $light;
+.rightCorner {
+  display: flex;
+  align-items: center;
+  flex: 0 0 auto;
+}
+
+.inputWrapper {
+  background-color: var(--input-background-colour, $white);
+  display: inline-flex;
+  font-size: $global-font-size;
+  padding: 0 20px;
+  width: var(--input-width, 100%);
+
+  input {
+    background-color: transparent;
+    flex-grow: 1;
+    border: none;
+    box-shadow: none;
   }
 }
 
-.shadedInput {
-  box-shadow: $form-field-shadow;
+.shadedInputWrapper {
   border: none;
-  padding: 0 15px;
-  height: 30px;
+  box-shadow: $form-field-shadow;
+  height: 25px;
```

Comment:
If the `size` property of the shaded input is optional; it is probably worth setting the height via a css variable:

 ```
height: var(--shaded-input-height, 25px); // or whatever is the default; can be overridden from the parent

&Large {
    --shaded-input-height: 40px; // so that it cannot be overridden from the parent
  }

&Small {
    --shaded-input-height: 30px;
  }

```

Reply:
✅